### PR TITLE
Use cqerl as cassandra client library

### PR DESCRIPTION
**Pull request by rslota**
This PR changes Cassandra backend to use cqerl client library instead seestar.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 language: erlang
+dist: trusty
```
Comment (fenek) [inline .travis.yml]: Why Trusty specifically?

Comment (michalwski) [inline .travis.yml]: Here's the answer: https://github.com/esl/MongooseIM/pull/1133#discussion_r94373464 :)

Comment (fenek) [inline .travis.yml]: Merci. :)

File: apps/ejabberd/src/mod_mam_cassandra_arch.erl [hunk 2]
```diff
@@ -501,13 +492,14 @@ from_id(ID, Filter = #mam_ca_filter{start_id = AfterID}) ->
...
     [row_to_uniform_format(Row) || Row <- MessageRows].
 
-row_to_uniform_format([MessID, BSrcJID, Data]) ->
-    SrcJID = unserialize_jid(BSrcJID),
-    Packet = stored_binary_to_packet(Data),
-    {MessID, SrcJID, Packet}.
+row_to_uniform_format(#{from_jid := FromJID, message := Msg, id := Id}) ->
+    SrcJID = unserialize_jid(FromJID),
+    Packet = stored_binary_to_packet(Msg),
+    MsgID = Id,
```
Comment (michalwski) [inline apps/ejabberd/src/mod_mam_cassandra_arch.erl]: Do we need `MsgID`? Can we simply return `Id` or assign `id` from the input map to `MsgID`?

Comment (rslota) [inline apps/ejabberd/src/mod_mam_cassandra_arch.erl]: No, it's just an artefact of refactoring. It'll be gone after next push.

**Comment by michalwski** [comment]
Thanks!

**Pull request merged**